File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/UndervaluedTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class UndervaluedTag extends Tag {

    public UndervaluedTag(int count) {
        super(count, Constants.UNDERVALUED_TAG, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/UndervaluedTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class UndervaluedTag(count: Int) : Tag(count, Constants.UNDERVALUED_TAG, Constants.UNDERVALUED_TAG_REMOTE, HashSet(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))